README.md

```diff
@@ -1285,6 +1285,7 @@ _Related: [Wikis](#wikis)_
 - [Spacebin](https://docs.spaceb.in) - Text-sharing for the final frontier — Reliable Pastebin server in Golang and Fiber. ([Demo](https://spaceb.in), [Source Code](https://github.com/spacebin-org/spirit)) `Apache-2.0` `Go`
 - [Stikked](https://github.com/claudehohl/Stikked) - Advanced and beautiful pastebin. ([Demo](https://paste.scratchbook.ch/)) `GPL-3.0` `PHP`
 - [Sup3rS3cretMes5age](https://github.com/algolia/sup3rS3cretMes5age) - Very simple (to deploy and to use) secret message service using Hashicorp Vault as a secrets storage. `MIT` `Go`
+- [Pastefy](https://pastefy.ga) - Beautiful, simple and easy to deploy Pastebin with optional Client-Encryption, Multitab-Pastes, an API, a highlighted Editor and more! ([Source Code](https://github.com/interaapps/pastefy), [Clients (IDEs/CLI)](https://github.com/topics/pastefy-addon), [Api-Clients](https://github.com/interaapps/accounts-java-client)) `MIT License` `Java`
```

Comment:
Hi,

- Only a single `Client` link is allowed. I suggest listing all related software in https://github.com/interaapps/pastefy/blob/master/README.md and linking to this instead
- `MIT License` -> `MIT`